extensions/capability-negotiation.md

```diff
@@ -176,6 +184,18 @@ The server MUST accept the `CAP` command at any time, including after registrati
 The LS subcommand is used to list the capabilities supported by the server.  The client
 should send an LS subcommand with no other arguments to solicit a list of all capabilities.
 
+The format of a server-sent `CAP LS` message is:
+
+    CAP LS <version>
```

Comment:
Should probably be `[<version>]` because it's an optional argument.

Reply:
Oops